File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/RaidProgression.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RaidProgression {

    @SerializedName("battle-of-dazaralor")
    @Expose
    public Raid battleOfDazaralor;
    @SerializedName("crucible-of-storms")
    @Expose
    public Raid crucibleOfStorms;
    @SerializedName("nyalotha-the-waking-city")
    @Expose
    public Raid nyalothaTheWakingCity;
    @SerializedName("the-eternal-palace")
    @Expose
    public Raid theEternalPalace;
    @SerializedName("uldir")
    @Expose
    public Raid uldir;

}

File: app/src/main/java/fr/jbme/raiderioapp/model/RIOCharacter/RaidProgression.kt
package fr.jbme.raiderioapp.model.RIOCharacter

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class RaidProgression(
    @SerializedName("battle-of-dazaralor")
    @Expose
    var battleOfDazaralor: Raid? = null,

    @SerializedName("crucible-of-storms")
    @Expose
    var crucibleOfStorms: Raid? = null,

    @SerializedName("nyalotha-the-waking-city")
    @Expose
    var nyalothaTheWakingCity: Raid? = null,

    @SerializedName("the-eternal-palace")
    @Expose
    var theEternalPalace: Raid? = null,

    @SerializedName("uldir")
    @Expose
    var uldir: Raid? = null
)